lib/markdown/snippet.js

```diff
@@ -24,7 +24,7 @@ module.exports = function snippet (md, options = {}) {
     const start = pos + 3
     const end = state.skipSpacesBack(max, pos)
     const rawPath = state.src.slice(start, end).trim().replace(/^@/, root)
-    const filename = rawPath.split(/[{\s]/).shift()
+    const filename = rawPath.split(/{/).shift()
```

Comment:
In that case, we need to add a `.trim()` because `rawPath` may be something in the form of `@/path/to/file {1}`.

Reply:
Ah, that's why it was also allowing a split on space -- sorry, I should have realized that myself!

I've updated the jest test I added so it checks the highlighting works when there's a space between the path & highlight as well.